### ISSUE TITLE
Error with openssl when running a Rscript

### ISSUE THREAD

**Issue by WPierrick** [jeroenooms/openssl#25]
Hi,

I get this error when I want to run a Rscript on a cluster where R is installed (the script works, and every user on the cluster has personal R libraries) :

Error in dyn.load(file, DLLpath = DLLpath, ...) : 
  unable to load shared object '/myfilepath/R/x86_64-pc-linux-gnu-library/3.2/openssl/libs/openssl.so':
  /myfilepath/R/x86_64-pc-linux-gnu-library/3.2/openssl/libs/openssl.so: symbol EC_KEY_get0_group, version OPENSSL_1.0.1_EC not defined in file libcrypto.so.10 with link time reference
Error: package or namespace load failed for 'missMethyl'

FYI, my version of openssl:
> packageVersion("openssl")
[1] ‘0.9.3’

Seems like it works when I manually load a package on R on a master head node of the cluster, but not when I run a job via bash on a cluster node.

Basically, R doesn't want to load any package because it seems there is a problem with incompatible openssl versions. Does that makes any sense? Any idea on how to solve this?
The problem arise after I tried to install ShinyMethyl, a bioconductor package.

Cheers,

**Comment by jeroenooms** [comment]
It looks like you have some conflict on the server with multile versions of libssl, where you compile against one version but then load another version. You might need to set `LD_LIBRARY_PATH` to the path from `pkg-config --libs openssl`.

**Comment by WPierrick** [comment]
Thank you for your reply!

There was indeed an incompatibility between openssl versions on my cluster's head node and  other nodes following a security update. It's been fixed now.
Thanks again.
